lib/registrant.rb

```diff
@@ -0,0 +1,28 @@
+class Registrant
+    attr_reader :age,
+                :license_data,
+                :name,
+                :permit
+
+    def initialize(name, age, permit = false)
+        @name = name
+        @age = age
+        @license_data = {
+            written: false,
+            license: false,
+            renewed: false
+            }
+        @permit = permit
+               
+    end
+
+    def permit?
+        @permit == @permit
```

Comment:
This would actually always give a `true`. You're comparing the same thing on both sides of the equation so there's no way this would ever give a `false`.  I'm curious what the tests look like for this method, will look in a second. We'd instead just want this method to be `@permit` inside so it's just reading the value of the instance variable.